### ISSUE TITLE
Clicking "reply" on a comment should shift focus to the form's textarea

### ISSUE THREAD

**Issue by benjiwheeler** [LLK/scratch-www#2668]
When I click "reply" on a comment, my instinct is to start typing right away -- that's what i'm used to from other interfaces.

But I can't start typing, until I click in the textarea to focus it.

I think we should focus the textarea as soon as it appears!

**Comment by WorldLanguages** [comment]
Yes please. This has been very annoying as a Scratch user.

**Comment by IamTregsthedev** [comment]
> Yes please. This has been very annoying as a Scratch user.

Yes Please from me too. It's just makes you mad sometimes.

**Comment by towerofnix** [comment]
Agreed! A similar topic would be the fact that there's no indication that the reply, send, etc buttons are focused (e.g. with tab), but I guess that's a separate issue.

**Comment by benjiwheeler** [comment]
Any outside contributor want to take this?
